src/NodaTime.Web/NodaTime.Web.csproj

```diff
@@ -24,7 +24,6 @@
     <PackageReference Include="morelinq" Version="2.10.0" />
   
     <!-- Handling of non-code folders -->
-    <Folder Include="Markdown\" />
```

Comment:
I assume this was redundant with the below?

Reply:
Apparently so. I didn't do that myself, so presumably VS was just being clever...